raiden-ts/tests/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "paths": {
-      "raiden-ts/*": ["../src/*"]
+      "@/*": ["../src/*"]
```

Comment:
While we are already on it: eventually add an alias for `<rootDir>/tests/`? Might be helpful to access shared mocks/generators/fixtures/...

Reply:
We don't do that many relative imports in the tests, and I'd rather have that matching the SDK pattern of using relative imports when possible. We could discuss this later in its own issue, if you wish, but I think we could leave it out of scope for this PR.

Reply:
Okay. Because for the dApp it will be definitely helpful.